yb-voyager/src/migassessment/sizing.go

```diff
@@ -1334,7 +1334,7 @@ func getReasoning(recommendation IntermediateRecommendation, shardedObjects []So
 
 	// Add information about colocated objects if they exist
 	if len(colocatedObjects) > 0 {
-		reasoning += fmt.Sprintf("%v objects (%v tables and %v explicit/implicit indexes) with %0.2f %v size "+
+		reasoning += fmt.Sprintf("%v objects (%v tables/materialized views and %v explicit/implicit indexes) with %0.2f %v size "+
```

Comment:
1. How/where are we ensuring that MVIEWs are getting considered for sizing.
2. Do MVIEW needs to considered separately or as TABLEs

Reply:
I verified this manually. Will work with Shubham on verifying any end to end tests